src/components/TextInput/TextInput.js

```diff
@@ -355,8 +355,19 @@ class TextInput extends React.Component<Props, State> {
       value: selectedValue,
     } = selector;
 
+    const shouldDisplaySpinner = this.getSelectorOptionsCount(selectorOptions) <= 1;
```

Comment:
why `<= 1`? sometimes we have just one option and it's fine, see aave withdraw or sablier withdraw